extensions/slack-status/src/defaultStatuses.ts

```diff
@@ -26,4 +26,14 @@ export const defaultStatuses: SlackStatusPreset[] = [
     emojiCode: ":walking:",
     defaultDuration: 0,
   },
+  {
+    title: "Working remotely",
```

Comment:
Two suggestions here: Use TitleCase (Working Remotely) and maybe you mean "Working from Home"? (not necessarily the same ;))

Reply:
Hi! TitleCase is not an issue. I'll change that.
Regarding working remotely or from home, I don't have a problem with both options. What do you prefer? I put remotely to have it 'open' but I don't have preferences. 😄

Reply:
I guess it depends on what your default way of working is because for the default you probably wouldn't set an explicit status anyway? (And then it also depends whether you generally work remotely, e.g. remote + in the home office 😅 ). I'm fine with having "Working from Home" and "Working from Office". One can always create new presets anyway ;).

Reply:
Sure! Let's keep it like that. I don't think is necessary to have a lot of presets.
Maybe in the future it could be good to be able to make custom presets for your own necessities. :)